Feature : Support LTR / RTL CSS styles

anbinm-msft opened the pull request.

msftclas commented:
Hi __@mttwc__, I'm your friendly neighborhood Microsoft Pull Request Bot (You can call me MSBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.microsoft.com.
    </span>

TTYL, MSBOT;

leakymemory commented:
Looks good.

anbiniyar committed:
Add support for LTR / RTL css styling

anbiniyar committed:
Fix inverted naming

This fixes the bug where.min.css was in fact not minified and .css was minified.

msftclas commented:
@anbinm-msft,
Thanks for your contribution as a Microsoft full-time employee or intern. You do not need to sign a CLA.
_Thanks,
Microsoft Pull Request Bot_

leakymemory reviewed (approved):
Awesome sauce.

mttwc commented:
🐑
